wapitiCore/attack/attack.py

```diff
@@ -230,6 +232,51 @@ def set_verbose(self, verbose):
     def set_color(self):
         self.color = 1
 
+    def add_payload(self, request_id: int, payload_type: str, category: str,
+    level=0, request=None, parameter="", info=""):
+        self.persister.add_payload(request_id, payload_type, self.name, category, level, request, parameter, info)
+
+    def add_payload_critical(self, request_id: int, payload_type: str, category: str,
```

Comment:
You can use [functools.partialmethod](https://docs.python.org/3/library/functools.html#functools.partialmethod) for all those methods :)